helm/datalab/charts/vault-auto-unseal/Chart.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: v2
+name: vault-auto-unseal
```

Comment:
Why did you choose to make a second chart for the vault-auto-unseal?

Reply:
I thought it might keep some of the separation that we currently have but I was just questioning this decision. I was thinking of moving this into a sub-directory within templates. Would that be the right way to do and maintain some structure? (I was also thinking configmaps etc. could then also be grouped in their own directory which could be nice)

Reply:
I'm trying to remember at what point in the deployment sequence do we need the vault to be unsealed. If it has to happen first then it's ok to keep it as a separate chart - my only thinking was whether to just make it part of the main chart with a conditional flag (as it's not needed in Azure or AWS where auto-unsealing is available)

Reply:
I think the later sounds like the right way to me as the unseal is just a Chron job that runs the whole time. It would be good to have to have it available conditionally too so I will have a look at that!